File: app/src/main/java/com/example/requestapp/ui/fragment/completedTasksFragment/CompletedTasksFragmentContract.java
package com.example.requestapp.ui.fragment.completedTasksFragment;

import com.example.requestapp.model.Task;

import java.util.List;

public interface CompletedTasksFragmentContract {

    interface Presenter {

        void completingTheList();

    }

    interface View {

        void showMessage(String message);

        void setTasksList(List< Task > tasksList);
    }
}

File: app/src/main/java/com/example/requestapp/ui/fragment/completedTaksksFragment/CompletedTasksFragmentContract.kt
package com.example.requestapp.ui.fragment.completedTaksksFragment

import com.example.requestapp.model.Task

interface CompletedTasksFragmentContract {
    interface Presenter {
        fun setListCompletedTask();
    }

    interface View {
        fun showMessage(message: String?)
        fun setTasksList(tasksList: List<Task?>?)
    }
}